tests/helpers/load_state_tests.py

```diff
@@ -520,6 +520,7 @@ def fetch_state_test_files(
 
 # Test case Identifier
 def idfn(test_case: Dict) -> str:
-    folder_name = test_case["test_file"].split("/")[-2]
-    # Assign Folder name and test_key to identify tests in output
-    return folder_name + " - " + test_case["test_key"]
+    if isinstance(test_case, dict):
```

Comment:
Does this need an else?

Reply:
If the `ids` function does not return an id, `pytest` will default to whatever id it was going to assign anyway. In our case, this scenario is applicable only when the `fetch_tests` function does not find any tests for that particular hardfork. In those cases, that test will be skipped anyway.